tests/pytests/perf_report_test.py

```diff
@@ -26,11 +26,11 @@
 # Canonical string generated from perf u-benchmarking scripts, used
 # to test out parsing methods.
 # pylint: disable=line-too-long
-MSG1 = 'Baseline - No logging, NumClients=5, NumOps=5000000 (5 Million), Server throughput=91795 (~91.79 K) ops/sec, Client throughput=20938 (~20.93 K) ops/sec, elapsed_ns=54469084570 (~54.46 Billion) ns'
+MSG1 = 'Baseline - No logging, NumClients=5, NumOps=5000000 (5 Million), NumThreads=1, Server throughput=91795 (~91.79 K) ops/sec, Client throughput=20938 (~20.93 K) ops/sec, elapsed_ns=54469084570 (~54.46 Billion) ns, NumOps/thread=5000 (5 K)'
```

Comment:
Update test msg-line format to conform to what's being spit out by the server in summary-line format mode.